### PR TITLE
Feat/loading

### LINKED ISSUE #4420: Feature: Loading indicator when tiddler is loading by loadTiddler()
When loading, tiddler's content will be blank, and without any indicator.

Is there a way to detect tiddler is being loading by SyncAdaptor, so I can add a loading indicator to it?

### PR DESCRIPTION
**Pull request by linonetwo**
closes #4420

图片

css is copied from https://css-loaders.com/

### CODE REVIEW

**Comment by straywriter** [comment]
oh my god, I desperately need this pr

File: core/ui/ViewTemplate/lazy-loading.tid [hunk 1]
```diff
@@ -0,0 +1,12 @@
+title: $:/core/ui/ViewTemplate/lazy-loading
+tags: $:/tags/ViewTemplate
+
+\whitespace trim
+\rules except dash
```
Comment (Jermolene) [inline core/ui/ViewTemplate/lazy-loading.tid:5]: Why is this needed?

File: themes/tiddlywiki/vanilla/base.tid [hunk 2]
```diff
@@ -1183,6 +1183,21 @@ button.tc-btn-invisible.tc-remove-tag-button {
 	color: <<colour muted-foreground>>;
 }
 
+.tc-tiddler-lazy-loading {
+	height: 4px;
+	width: 100%;
+	--c:no-repeat linear-gradient(<<colour background>> 0 0);
```
Comment (Jermolene) [inline themes/tiddlywiki/vanilla/base.tid:1189]: The core does not currently use CSS variables; can we avoid them here?

**Review by Jermolene — COMMENTED** [review]
Thanks @linonetwo

File: core/ui/ViewTemplate/lazy-loading.tid (hunk 1 above)
Comment (linonetwo) [reply core/ui/ViewTemplate/lazy-loading.tid:5]: Because it break the `--c` CSS variables. Let me remove this.

File: themes/tiddlywiki/vanilla/base.tid (hunk 2 above)
Comment (linonetwo) [reply themes/tiddlywiki/vanilla/base.tid:1189]: I copied this from https://css-loaders.com/ , I'm not good at writing styles.

**Comment by linonetwo** [comment]
Tested with 

```
    "server:lazy": "node ./tiddlywiki.js ./editions/tw5.com-server --listen root-tiddler=$:/core/save/lazy-all",
```

**Comment by Jermolene** [comment]
Thanks for your patience @linonetwo
